featuretools/computational_backends/feature_set_calculator.py

```diff
@@ -117,33 +117,32 @@ def progress_callback(*args):
 
         if len(df.columns) == 0:
             default_df = self.generate_default_df(instance_ids=instance_ids)
-            if isinstance(df, dd.core.DataFrame):
+            if isinstance(df, dd.DataFrame):
                 cols = [col for col in default_df.columns] + [default_df.index.name]
                 return dd.from_pandas(default_df.reset_index(), npartitions=1)[cols]
             return default_df
```

Comment:
It looks like none of these lines are hit by our current tests - including the pandas code on line 119. Not sure what scenario causes this to be necessary, but wondering if the Dask code block is necessary?

Reply:
Given that we don't allow empty Dask EntitySets anyways, I have a hard time imagining we could ever hit this point (a Dask dataframe with no columns) but I wasn't sure if it was a valid edge case

Reply:
Seems like it could be an edge case since the pandas code is there. My inclination would be to remove at least the Dask code until we know for sure we need it with an example that we can create a test with so we would have the edge case documented going forward.